site/docs/v1/tech/oauth/tokens.adoc

```diff
@@ -103,6 +103,10 @@ https://tools.ietf.org/html/rfc7519#section-4.1.1[RFC 7519 Section 4.1.1].
 [field]#permissions# [type]#[Array<String>]#::
 The permission granted to the recipient Entity by the target Entity. This claim is only present if permissions are associated with the grant and any requested permissions are found in the grant.
 
+[field]#scope# [type]#[String]#::
```

Comment:
We handle scopes a bit differently than other providers. Usually a scope is either a standardized string (like `offline_access`, defined in the OIDC spec) or custom, defined by the user of the idp (as outlined here: https://github.com/FusionAuth/fusionauth-issues/issues/275 ). But for this case, we have a specific format we support.

Can you add a link to https://fusionauth.io/docs/v1/tech/oauth/#client-credentials-scopes which outlines what scope values are valid?